### ISSUE TITLE
Add "none" option for LoRA loader/related

### ISSUE THREAD

**Issue by space-nuko** [comfyanonymous/ComfyUI#626]
It would be nice to have a `None` option for the LoRA loader that makes it act like a passthrough.

**Comment by comfyanonymous** [comment]
Setting the strengths to zero will also do that.

**Comment by space-nuko** [comment]
Well also there could be the case where the user has no LoRAs. Does the node work with no LoRAs to pick from?

**Comment by WASasquatch** [comment]
> Setting the strengths to zero will also do that.

That's also two clicks and num pads plus enter away, or fiddling with precision of the incremental buttons via holding down or numerous clicks. That's a lot of work over just two clicks to select a none

**Comment by fractal-fumbler** [comment]
> > Setting the strengths to zero will also do that.
> 
> That's also two clicks and num pads plus enter away, or fiddling with precision of the incremental buttons via holding down or numerous clicks. That's a lot of work over just two clicks to select a none. If none just pass the model/clip as it came in.

i've also found out that clicking LMB on the field line and while holding LMB moving right or left changes value in field

tho i agree that none for LoRA is easier to choose and more intuitive than setting strength to zero

**Comment by brknsoul** [comment]
Select the Lora Loader and hit Ctrl+B to bypass it.

**Comment by WASasquatch** [comment]
> Select the Lora Loader and hit Ctrl+B to bypass it.

It's not sufficient for the API is it? You may just want to be passing LORAs as a filename from paths.

**Comment by StephanGocht** [comment]
> It's not sufficient for the API is it? You may just want to be passing LORAs as a filename from paths.

Indeed, setting the LORA to none would be convenient. However it isn't too difficult to skip/bypass nodes programmatically (where prompt contains the loaded API json file):

```python
import copy

def input_is_from_node(value, node_id):
    return isinstance(value, list) \
        and len(value) == 2 \
        and value[0] == node_id

def bypass_node(prompt, bypassed_node_id):
    prompt = copy.deepcopy(prompt)
    bypassed_node = prompt[bypassed_node_id]
    del prompt[bypassed_node_id]

    for node in prompt.values():
        for key, value in node['inputs'].items():
            if input_is_from_node(value, bypassed_node_id):
                node['inputs'][key] = bypassed_node['inputs'][key]

    return prompt
```

**Comment by pileon** [comment]
> Well also there could be the case where the user has no LoRAs. Does the node work with no LoRAs to pick from?

This is kind of important for my use-case, where I want to embed the LoRA inside a sub-graph and only expose the LoRA to select. If I don't want to use the LoRA it would be extremely convenient to set to "none" instead of selecting a dummy LoRA and set strengths to zero.
